packages/delivery-expo/network-status.js

```diff
@@ -24,11 +24,11 @@ module.exports = class NetworkStatus {
 
   _watch () {
     // get the initial status
-    NetInfo.isConnected.fetch().then(isConnected => {
-      this._update(isConnected)
+    NetInfo.fetch().then(state => {
+      this._update(state.isInternetReachable)
       // then listen for subsequent changes
-      NetInfo.isConnected.addEventListener('connectionChange', isConnected => {
-        this._update(isConnected)
+      NetInfo.addEventListener(state => {
+        this._update(state.isInternetReachable)
```

Comment:
Is a change to use `isInternetReachable` (as opposed to `isConnected`) the correct state to check? For example, a device on a private network using an On Premise installation presumably wouldn't have internet connectivity but we'd still want to trigger a change of status.

Reply:
Hmm good point – I'll revert to using `isConnected`.